memento/src/test/java/com/alexstyl/specialdates/events/DateTest.java

```diff
@@ -40,6 +40,27 @@ public void givenAEndOfTheYearDate_whenAddingOneDay_thenTheFirstDayOftheNextYear
         assertThat(firstDayOfNextYear.getYear()).isEqualTo(nextYear);
     }
 
+    @Test
+    public void givenDateWithShortMonth_thenReturn29Days() {
```

Comment:
`givenDateWithShortMonthAndNoYearSpecified_thenReturn29Days` since the year is an important factor.